configs/wrapper_config.yaml

```diff
@@ -0,0 +1,13 @@
+---
+novaseq:
+  root_path: "/seqstore/instruments/novaseq_687_gc/"
+  demultiplex_path: "/seqstore/instruments/novaseq_687_gc/Demultiplexdir/"
+
+previous_runs_file_path: 'novaseq_runlist.txt'
```

Comment:
Where will this file be saved? in the repo?

Reply:
yes it's saved in the repo